mirakuru/http.py

```diff
@@ -50,16 +53,21 @@ def __init__(self, command, url, **kwargs):
         """
         An :func:`urlparse.urlparse` representation of an url.
 
-        It'll be used to check process status on."""
+        It'll be used to check process status on.
+        """
+
+        port = self.url.port
```

Comment:
it seems to be a common practice to use either `x = y or Z` if any false is false, or `x = y if condition else Z` so as long as it's not long expression, it's okay to assign defaults like that. And since I don't think we could expect port 0 being used, the or idiom would be quite in place here.